tests/test_w3tagb.f90

```diff
@@ -0,0 +1,15 @@
+program test_w3tagb
+  implicit none
+  character(len=:), allocatable :: prog, org
+  integer :: year, julian_day, hundreths_of_julian_day
+
+  prog = "test_w3tagb"
```

Comment:
Add a print statement like:

print *, 'Testing wetagb...' at the beginnning, to match other test output.